manual/source/mpes-structure.rst

```diff
@@ -4,18 +4,23 @@
 Photoemission & core-level spectroscopy
 =======================================
 
-The NXmpes application definition aims at describing data and metadata obtained with any possible multidimensional PES setup.
-Hence, it is very general and currently the only limitation to the data is that an energy
-axis has to be present.
-The general experimental techniques described by this application definitions are
-photon-in and photoelectron-out.
-If you want NXmpes for such a techniques, changes are good you find something useful here.
+The NXmpes application definition is designed to describe data and metadata obtained from
+various multidimensional photoemission spectroscopy (mpes) techniques.
+It is highly versatile, with the only requirement being the presence of an energy axis.
```

Comment:
We require many more things than just the energy axis, right? Maybe we can design the old and new rst file to say: "It is highly versatile, with the only requirement on the data being the presence of an energy axis."

Reply:
True, this might come over a bit too lax on the actual constraints. I don't know if I'm fully happy with data, as one could also interpret data as the whole data inside the file (not knowing that there is the data/NXdata field).

Maybe something like
```
This tool is very flexible and requires just a reasonable amount of metadata to make the stored data interoperable and reproducible.
The only thing needed for the actual data is the presence of an energy axis.
```

which makes it (hopefully) a bit clearer with the distinguishing between metadata and data?

Reply:
Ok, sounds good. Maybe to avoid using the generic term _thing_, you could replace it by _element_ or you say: "The only requirement for the actual data is the existence of an energy axis."

Reply:
I used your suggestion @lukaspie. With this I'm resolving this comment.